vcpkg.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "openloco",
-  "version-string": "22.02",
+  "version-string": "25.03",
```

Comment:
I think we should just zero this one out, this will be never used and its very unlikely we'll ever have a vcpkg library port.

Reply:
Looking at the [vcpkg.json reference page](https://learn.microsoft.com/en-us/vcpkg/reference/vcpkg-json#version), `version-string` can be an arbitrary string, so I suppose an empty string would go. However, that seems against the spirit of having a mandatory version field.

We could switch to `version-date` and just have "a date formatted as YYYY-MM-DD" there. That would remove any semblance of OpenLoco version numbers here. 

Thoughts?